new_lineage_panel/src/CustomNodes.tsx

```diff
@@ -80,6 +81,7 @@ export const NodeTypeIcon: FunctionComponent<{ nodeType: string }> = ({
     {nodeType === "model" && <ModelIcon />}
     {nodeType === "source" && <SourceIcon />}
     {nodeType === "exposure" && <ExposureIcon />}
+    {nodeType === "analysis" && <AnalysisIcon />}
```

Comment:
not in scope of this PR. but it is better to have defined enums for these strings